docs/custom_resources.adoc

```diff
@@ -205,43 +232,53 @@ status:
+[discrete]
+=== spec.status
 
-=== Status
+`updated`:: ISO-formatted timestamp that shows when the status was created
+`serviceAccountSecretName`:: Secret name that contains the credentials for the service account
 
- * `updated` - an ISO timestamp of when the status was created
- * `serviceAccountSecretName` - the secret name contained the credentials for the service account.
+[discrete]
+=== spec.status.conditions
 
+`ServiceAccountCreated`:: If the value of this field is `True`, the Operator successfully created the service account.
+`ServiceAccountSecretCreated`:: If the value of this field is `True`, the Operator successfully created the credentials secret.
+`Finished`:: Indicates whether the Operator has successfully finished processing the CR. For more information, see xref:finished-condition[Finished condition].
+`AcccesTokenSecretValid`:: Indicates whether the token specified as a value for the `accessTokenSecretName` field was available and successfully exchanged with the authentication service. For more information, see xref:access-token-secret-valid[AcccesTokenSecretValid condition].
 
-==== Conditions
- 
- * ServiceAccountCreated - When status is `True` then the service account was created successfully.
- * ServiceAccountSecretCreated - When status is `True` then the credentials secret was created successfully.
- * Finished - see `Finished Condition`
- * AcccesTokenSecretValid - see `AcccesTokenSecretValid Condition`
+== Conditions common to all RHOAS Custom Resources
 
+Each CR type managed by the RHOAS Operator has several conditions defined in the `status` subresource. Some conditions are shared by multiple types and some are type-specific.
 
-== Conditions Common to RHOAS Custom Resources
+All conditions have a `status` value of `True`, `False`, or `Unknown`. When the Operator processes a CR, it first sets the `status` value of each condition to `Unknown`. Then, the Operator sets each condition to a value of `True` as the condition is checked. If a condition fails to check, the Operator sets the `status` values of the condition *and* the `Finished` condition to `False`. The Operator also updates the `reason` and `message` fields with more information.
 
-Each custom resource type has several conditions on their status subresource. Some conditions are shared by multiple types and some are type specific. All types have the status of `True`, `False`, and `Unknown`. When the operator begins running on the resource, it will set all conditions to `Unknown`. The operator then sets each condition to `True` as they are checked. If a condition fails to check, it and the `Finished` conditions will be set to `False` as well as have their `reason` and `message` fields set.  Additionally, processing is halted and subsequent conditions are not checked and they will remain `Unknown`. In order to resume processing, the errors must be corrected and a new custom resource submitted.
```

Comment:
I have feeling that we missing some info that was removed here.

Reply:
@wtrocki I think you mean info removed from the current version in the 'app-services-operator' repo, and *not* from the PR I opened? I can't see anything missing from my updated version when I compare against the current version.